scripts/deactivate_iac.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+import argparse
+import logging
+import os
+import sys
+
+import requests
+from structlog import configure, wrap_logger
+from structlog.processors import TimeStamper, JSONRenderer
+from structlog.stdlib import add_log_level, filter_by_level, LoggerFactory
+
+# Script needs to know about app so append to PYTHONPATH
+sys.path.append(os.path.join(os.path.dirname(__file__), os.path.pardir))
+from app import config  # NOQA
+
+
+try:
+    config_info = getattr(config, os.environ['APP_SETTINGS'])
+except (AttributeError, KeyError) as e:
+    config_info = config.DevelopmentConfig
+
+# Put config into a dict
+config = dict((name, getattr(config_info, name)) for name in dir(config_info) if not name.startswith('__'))
+
+# Configure logger
+logger = wrap_logger(logging.getLogger(__name__))
+
+# Configure argument parser
+parser = argparse.ArgumentParser(description='Deactivate IACs for a Collection Exercise.')
+parser.add_argument('-c', dest='collectionExercise', type=str, help='Collection Exercise ID.')
+
+# Setup URLs
+case_url = f"{config['CASE_URL']}/cases/"
+iac_url = f"{config['IAC_URL']}/iacs/"
+ci_url = f"{config['COLLECTION_INSTRUMENT_URL']}/collection-instrument-api/1.0.2/collectioninstrument/id/"
```

Comment:
doesn't look like this is being used so should be removed

Reply:
Removed.